### DISCUSSION TITLE
Erro csharp(CS8604)

### DISCUSSION THREAD

**Discussion by GustavoBaco** [balta-io/2801#31] [Perguntas e Respostas] [answered]
Boa tarde, professor. 
Estou tendo um pequeno problema que não estou sabendo resolver
![image](https://user-images.githubusercontent.com/89161781/150209725-fd80d1db-f79a-44dd-a8e1-bb9a61f9fac4.png)

Como posso resolver esse problema ?

**Comment by andrebaltieri** [comment, marked as answer]
Boa noite, @GustavoBaco como vai?

Este aviso é por conta do método `Console.ReadLine();` que pode retornar um valor nulo.
Ele não impede a execução do seu programa, é apenas um aviso :)

**Reply by GustavoBaco** [reply]
Obrigado, professor @andrebaltieri .
